soci/artifacts_test.go

```diff
@@ -98,14 +98,41 @@ func TestGetIndexArtifactEntries(t *testing.T) {
 	}
 }
 
+func TestArtifactDbPath(t *testing.T) {
+	tests := []struct {
+		name     string
+		root     string
+		expected string
+	}{
+		{
+			name:     "default",
+			root:     "",
+			expected: "/var/lib/soci-snapshotter-grpc/artifacts.db",
+		},
+		{
+			name:     "custom",
+			root:     "/tmp",
```

Comment:
Isn't /tmp in memory? Any reason we can't just use a tempdir or something? Just want to make sure it gets cleaned up is all.

Reply:
Oh, `ArtifactsDbPath` doesn't actually create any files on disk. It is just some logic to append `artifacts.db` to the provided path or default to `/var/lib/soci-snapshotter-grpc/artifacts.db` if the provided path is empty.

Reply:
Makes sense, might be worth clearing it up with a different folder name but not worth blocking, LGTM